lib/cli.js

```diff
@@ -118,6 +118,10 @@ exports.run = () => {
 		throw new Error(colors.error(figures.cross) + ' Watch mode is not available in CI, as it prevents AVA from terminating.');
 	}
 
+	if (hasFlag('--concurrency') || hasFlag('-c')) {
+		throw new Error(colors.error(figures.cross) + ' The --concurrency and -c flags must be provided the maximum number of test files to run with.');
```

Comment:
`the maximum number of test files to run with.` => `the maximum number of test files to run at once.`